org.gpodder.gpodder-adaptive.json

```diff
@@ -0,0 +1,52 @@
+{
+  "app-id": "org.gpodder.gpodder-adaptive",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "42",
+  "sdk": "org.gnome.Sdk",
+  "command": "gpodder",
+  "rename-icon": "gpodder-adaptive",
+  "copy-icon": true,
+  "rename-desktop-file": "gpodder.desktop",
+  "finish-args": [
+    "--device=dri",
+    "--socket=x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--share=network",
+    "--own-name=org.gpodder.*",
```

Comment:
`org.gpodder.gpodder-adaptive` is granted automatically. Why do you need this

Reply:
This is to make `org.gpodder.gpodder` and this package co-installable, but not able to start at the same time (both versions claim org.gpodder.gpodder as application-id).

I'm not sure if `--own-name=org.gpodder.gpodder` would be enough though.

Reply:
hence what I said about being the same app.